src/kc-app/src/keymaster-sdk.js

```diff
@@ -30,7 +30,7 @@ export async function saveWallet(wallet) {
 
 export async function newWallet(mnemonic, overwrite = false) {
     try {
-        const response = await axios.post(`/api/v1/wallet`, { mnemonic, overwrite });
+        const response = await axios.post(`/api/v1/wallet/new`, { mnemonic, overwrite });
```

Comment:
Your version seems to be behind https://github.com/KeychainMDIP/kc/blob/main/src/kc-app/src/keymaster-sdk.js

Reply:
sync issue resolved 👍